core/src/test/java/org/elasticsearch/indices/memory/IndexingMemoryControllerIT.java

```diff
@@ -109,6 +110,45 @@ public void testIndexBufferSizeUpdateInactiveShard() throws InterruptedException
                             shard1.engine().config().getIndexingBufferSize().bytes() + "]"
             );
         }
+
+        // Make sure we also pushed the tiny indexing buffer down to the underlying IndexWriter:
+        assertEquals(EngineConfig.INACTIVE_SHARD_INDEXING_BUFFER.bytes(), getIWBufferSize("test1"));
+    }
+
+    private long getIWBufferSize(String indexName) {
+        return client().admin().indices().prepareStats(indexName).get().getTotal().getSegments().getIndexWriterMaxMemoryInBytes();
+    }
+
+    @Test
+    public void testIndexBufferSizeTwoShards() throws InterruptedException {
+        createNode(Settings.builder().put(IndexingMemoryController.SHARD_INACTIVE_TIME_SETTING, "100000h",
```

Comment:
> do we want to speed up indices.memory.interval as well?

I think it's not needed here: the test is just confirming that on init, the setting was parsed correctly (and not that e.g. the memory controller work up and did something important).

In fact: I don't know why the other test must set `100ms` since `createNode` does it as well ... I'll try to remove it!